### ISSUE TITLE
Accuracy suddenly drops to 0% during training

### ISSUE THREAD

**Issue by luxifeo** [PaddlePaddle/PaddleOCR#3254]
Hi all. I am trying to train a custom recognition model. The process seams to work normally, then the loss suddenly increases to very high value and accuracy drops to zero. 

Here is the output log.

> [2021/07/04 02:49:04] root INFO: epoch: [9/40], iter: 202600, lr: 0.009126, loss: 0.795570, acc: 0.851562, norm_edit_dis: 0.982678, reader_cost: 0.00215 s, batch_cost: 0.14965 s, samples: 12800, ips: 855.35216
> [2021/07/04 02:49:28] root INFO: epoch: [9/40], iter: 202700, lr: 0.009125, loss: 0.859082, acc: 0.843750, norm_edit_dis: 0.980979, reader_cost: 0.00185 s, batch_cost: 0.14933 s, samples: 12800, ips: 857.18863
> [2021/07/04 02:49:53] root INFO: epoch: [9/40], iter: 202800, lr: 0.009124, loss: 0.968477, acc: 0.835938, norm_edit_dis: 0.977872, reader_cost: 0.00142 s, batch_cost: 0.14883 s, samples: 12800, ips: 860.01396
> [2021/07/04 02:50:18] root INFO: epoch: [9/40], iter: 202900, lr: 0.009123, loss: 0.801612, acc: 0.843750, norm_edit_dis: 0.982128, reader_cost: 0.00117 s, batch_cost: 0.14836 s, samples: 12800, ips: 862.78098
> [2021/07/04 02:50:42] root INFO: epoch: [9/40], iter: 203000, lr: 0.009122, loss: 0.718845, acc: 0.859375, norm_edit_dis: 0.983759, reader_cost: 0.00126 s, batch_cost: 0.14838 s, samples: 12800, ips: 862.66448
> [2021/07/04 02:50:44] root INFO: cur metric, acc: 0.8426573426573427, norm_edit_dis: 0.9505307509825608, fps: 1389.277823083331
> [2021/07/04 02:50:44] root INFO: best metric, acc: 0.8826173826173827, norm_edit_dis: 0.9663079545259894, fps: 1338.1706286966241, best_epoch: 8
> [2021/07/04 02:51:09] root INFO: epoch: [9/40], iter: 203100, lr: 0.009121, loss: 0.760324, acc: 0.859375, norm_edit_dis: 0.982939, reader_cost: 0.00172 s, batch_cost: 0.14916 s, samples: 12800, ips: 858.14298
> [2021/07/04 02:51:34] root INFO: epoch: [9/40], iter: 203200, lr: 0.009120, loss: 0.687531, acc: 0.859375, norm_edit_dis: 0.984275, reader_cost: 0.00170 s, batch_cost: 0.14887 s, samples: 12800, ips: 859.83074
> [2021/07/04 02:51:59] root INFO: epoch: [9/40], iter: 203300, lr: 0.009119, loss: 0.719847, acc: 0.859375, norm_edit_dis: 0.983978, reader_cost: 0.00167 s, batch_cost: 0.14877 s, samples: 12800, ips: 860.39078
> [2021/07/04 02:52:23] root INFO: epoch: [9/40], iter: 203400, lr: 0.009118, loss: 0.730239, acc: 0.867188, norm_edit_dis: 0.983633, reader_cost: 0.00153 s, batch_cost: 0.14860 s, samples: 12800, ips: 861.39765
> [2021/07/04 02:52:48] root INFO: epoch: [9/40], iter: 203500, lr: 0.009117, loss: 0.825789, acc: 0.859375, norm_edit_dis: 0.983094, reader_cost: 0.00142 s, batch_cost: 0.14852 s, samples: 12800, ips: 861.84235
> [2021/07/04 02:52:50] root INFO: cur metric, acc: 0.8336663336663337, norm_edit_dis: 0.9479489748237743, fps: 1380.3273775561013
> [2021/07/04 02:52:50] root INFO: best metric, acc: 0.8826173826173827, norm_edit_dis: 0.9663079545259894, fps: 1338.1706286966241, best_epoch: 8
> [2021/07/04 02:53:15] root INFO: epoch: [9/40], iter: 203600, lr: 0.009117, loss: 0.741417, acc: 0.843750, norm_edit_dis: 0.982400, reader_cost: 0.00236 s, batch_cost: 0.14982 s, samples: 12800, ips: 854.34330
> **[2021/07/04 02:53:39] root INFO: epoch: [9/40], iter: 203700, lr: 0.009116, loss: 0.776616, acc: 0.859375, norm_edit_dis: 0.982723, reader_cost: 0.00197 s, batch_cost: 0.14930 s, samples: 12800, ips: 857.33725**
> **[2021/07/04 02:54:04] root INFO: epoch: [9/40], iter: 203800, lr: 0.009115, loss: 63.319569, acc: 0.000000, norm_edit_dis: 0.019041, reader_cost: 0.00192 s, batch_cost: 0.14927 s, samples: 12800, ips: 857.49274**
> [2021/07/04 02:54:28] root INFO: epoch: [9/40], iter: 203900, lr: 0.009114, loss: 58.835430, acc: 0.000000, norm_edit_dis: 0.004510, reader_cost: 0.00162 s, batch_cost: 0.14895 s, samples: 12800, ips: 859.34120
> [2021/07/04 02:54:53] root INFO: epoch: [9/40], iter: 204000, lr: 0.009113, loss: 57.744995, acc: 0.000000, norm_edit_dis: 0.018229, reader_cost: 0.00178 s, batch_cost: 0.14916 s, samples: 12800, ips: 858.14497
> **[2021/07/04 02:54:55] root INFO: cur metric, acc: 0.0, norm_edit_dis: 0.03144527904508354, fps: 1390.9981178364894**
> [2021/07/04 02:54:55] root INFO: best metric, acc: 0.8826173826173827, norm_edit_dis: 0.9663079545259894, fps: 1338.1706286966241, best_epoch: 8
> [2021/07/04 02:55:19] root INFO: epoch: [9/40], iter: 204100, lr: 0.009112, loss: 55.408508, acc: 0.000000, norm_edit_dis: 0.045044, reader_cost: 0.00238 s, batch_cost: 0.14984 s, samples: 12800, ips: 854.22757
> [2021/07/04 02:55:44] root INFO: epoch: [9/40], iter: 204200, lr: 0.009111, loss: 54.524658, acc: 0.000000, norm_edit_dis: 0.069823, reader_cost: 0.00188 s, batch_cost: 0.14918 s, samples: 12800, ips: 857.99606
> [2021/07/04 02:56:09] root INFO: epoch: [9/40], iter: 204300, lr: 0.009110, loss: 52.647797, acc: 0.000000, norm_edit_dis: 0.089146, reader_cost: 0.00168 s, batch_cost: 0.14907 s, samples: 12800, ips: 858.65538
> [2021/07/04 02:56:33] root INFO: epoch: [9/40], iter: 204400, lr: 0.009109, loss: 60.678497, acc: 0.000000, norm_edit_dis: 0.019597, reader_cost: 0.00179 s, batch_cost: 0.14897 s, samples: 12800, ips: 859.24308
> [2021/07/04 02:56:57] root INFO: epoch: [9/40], iter: 204500, lr: 0.009108, loss: 59.000465, acc: 0.000000, norm_edit_dis: 0.018042, reader_cost: 0.00165 s, batch_cost: 0.14901 s, samples: 12800, ips: 858.98813

My training config

> Global:
>   debug: false
>   use_gpu: true
>   epoch_num: 40
>   log_smooth_window: 100
>   print_batch_step: 100
>   save_model_dir: ./output/rec
>   save_epoch_step: 5
>   eval_batch_step:
>   - 0
>   - 500
>   cal_metric_during_train: true
>   pretrained_model: null
>   checkpoints: null
>   save_inference_dir: null
>   use_visualdl: false
>   infer_img: doc/imgs_words/ch/word_1.jpg
>   character_dict_path: configs/dict/chrVN260.txt
>   character_type: latin
>   max_text_length: 150
>   infer_mode: false
>   use_space_char: true
>   save_res_path: ./output/rec/predicts.txt
>   distributed: false
> Optimizer:
>   name: Adam
>   beta1: 0.9
>   beta2: 0.999
>   lr:
>     name: CyclicalCosine
>     learning_rate: 0.01
>     warmup_epoch: 1
>     cycle: 40
>   regularizer:
>     name: L2
>     factor: 1.0e-05
> Architecture:
>   model_type: rec
>   algorithm: CRNN
>   Transform: null
>   Backbone:
>     name: MobileNetV3
>     scale: 0.5
>     model_name: small
>     small_stride:
>     - 1
>     - 2
>     - 2
>     - 2
>   Neck:
>     name: SequenceEncoder
>     encoder_type: rnn
>     hidden_size: 48
>   Head:
>     name: CTCHead
>     fc_decay: 1.0e-05
> Loss:
>   name: CTCLoss
> PostProcess:
>   name: CTCLabelDecode
> Metric:
>   name: RecMetric
>   main_indicator: acc
> Train:
>   dataset:
>     name: LMDBDataSet
>     data_dir: lmdb/train
>     transforms:
>     - DecodeImage:
>         img_mode: BGR
>         channel_first: false
>     - RecAug: null
>     - CTCLabelEncode: null
>     - RecResizeImg:
>         image_shape:
>         - 3
>         - 32
>         - 640
>     - KeepKeys:
>         keep_keys:
>         - image
>         - label
>         - length
>   loader:
>     shuffle: true
>     batch_size_per_card: 128
>     drop_last: true
>     num_workers: 6
> Eval:
>   dataset:
>     name: LMDBDataSet
>     data_dir: lmdb/val
>     transforms:
>     - DecodeImage:
>         img_mode: BGR
>         channel_first: false
>     - CTCLabelEncode: null
>     - RecResizeImg:
>         image_shape:
>         - 3
>         - 32
>         - 640
>     - KeepKeys:
>         keep_keys:
>         - image
>         - label
>         - length
>   loader:
>     shuffle: false
>     drop_last: false
>     batch_size_per_card: 128
>     num_workers: 6
> 

I am using  paddlepaddle-gpu==2.0.0 and latest code at branch release/2.1, hardware is RTX 2080 Ti.

**Comment by littletomatodonkey** [comment]
Seems that lr (0.01) is too large for the bs 128,  0.0002 might be a better choice if you just use single gpu card.

By the way, `CyclicalCosine`  is as follows, for which loss and acc might change a lot if your lr is not set properly.

![image](https://user-images.githubusercontent.com/14270174/124534070-1a954800-de46-11eb-94e5-feae3dafb37c.png)

**Comment by luxifeo** [comment]
Thank you for your suggestion. I will try to lower the learning rate and retrain. Hope this will fix my problem.
